### ISSUE TITLE
Download Helper tests print log to screen

### ISSUE THREAD

**Issue by sabjorn** [sabjorn/BandcampEnhancementSuite#48]
Even though:

```
// Prevent Logger output during tests
sandbox.stub(dh, "log");
```

is set in the tests, the logger still prints to screen when running tests

**Comment by emcniece** [comment]
I recall messing around with this to properly silence the logger. `winston` and its dependencies accounts for a significant portion of the bundle, and it's annoying to fight through this silencing - I propose we remove Winston and implement a very basic logging class ourselves.

**Comment by sabjorn** [comment]
fixed with 19375a1a1b36860cb8d450f5c95ad989b302145b
